src/index.ts

```diff
@@ -0,0 +1,41 @@
+import {Boom} from '@hapi/boom';
+import {Request, Response} from 'express';
+
+export class BoomableError extends Error {
+  public statusCode = 500;
+}
+
+type ExpressErrorMiddleware = (
+  err: Boom | BoomableError | Error,
+  req: Request,
+  res: Response,
+  next: (err: Error) => void,
+) => void;
+
+export default function (): ExpressErrorMiddleware {
+  return (
+    err: Boom | BoomableError | Error,
+    _req: Request,
+    res: Response,
+    _next: (err: Error) => void,
+  ) => {
+    if (!(err instanceof Error)) {
+      err = new Error(err);
+    }
+    let boom: Boom;
+    if (err instanceof Boom) {
+      boom = err;
+    } else {
+      let statusCode = 500;
+      if (err instanceof BoomableError) {
+        statusCode = err.statusCode ?? 500;
+      }
+      boom = new Boom(err, {statusCode});
+    }
+    boom.reformat();
+    res.set(boom.output.headers);
+    res.status(boom.output.statusCode);
+    const body = boom.output.payload;
```

Comment:
This line was previously `res.body =`. This was a very intentional behavior (flashback! https://github.com/goodeggs/crashpad/pull/2). Unfortunately it looks like it didn't have test coverage.

It's a pretty weird behavior that we should probably remove in a future breaking version. I think this would be an OK use of a `@ts-ignore`. 😬

Consider trying to add the missing test coverage!